src/app/components/MediaLoader/configs/liveMedia.ts

```diff
@@ -37,9 +37,12 @@ export default ({
         holdingImageURL,
         items: [
           {
-            versionID: video?.vpid,
+            ...(video.serviceId && { serviceID: video.serviceId }),
+            ...(!video.serviceId && {
+              versionID: video?.vpid,
```

Comment:
Is there a case when we're not expecting the video object to exist? From this `video?.vpid` it looks like we are, if so we should guard both spreads with optional chaining, then you can get rid of the internal optional chaining. What might be better is destructuring the video object in the function body

Reply:
Good spot! Let me refactor / tidy this up a bit